src/style-spec/reference/v8.json

```diff
@@ -2566,7 +2567,10 @@
         "group": "Types",
         "sdk-support": {
           "basic functionality": {
-            "js": "0.48.0"
+            "js": "0.48.0",
+            "android": "6.7.0",
+            "ios": "4.6.0",
+            "macos": "0.12.0"
```

Comment:
The iOS and macOS do support the `format` operator in style JSON and at runtime via `MGL_FUNCTION` in NSExpression. mapbox/mapbox-gl-native#12739 tracks full support via an idiomatic NSExpression syntax.

As a general rule, we agreed in https://github.com/mapbox/mapbox-gl-js/pull/6988#discussion_r203505507 to leave out an entry when the SDK supports a property in style JSON but not fully at runtime. But Studio relies on this file for its compatibility options, so I’m coming around to the idea that we should go ahead and acknowledge support for a property even if it’s only supported in style JSON. Otherwise, Studio users could be prevented from using a property until it gets runtime styling support, which seems unfair.

/ref @tobrun